test/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner_test.rb

```diff
@@ -274,4 +289,15 @@
       end
     end
   end
+
+  describe ".check_system_dependencies!" do
+    subject { as_task_runner.check_system_dependencies! }
+
+    it "should call the methods" do
+      as_task_runner.expects(:check_tool_version!).with("npm", "5.2.0")
+      as_task_runner.expects(:check_tool_version!).with("node", "14.5.0")
```

Comment:
I don't think `.expect` on its own methods is valuable in unit testing.  `.stubs` on `TaskRunner` and not `ASTaskRunner` makes sense to me.  Again, WDYT?